tests/test_ext_marshmallow_field.py

```diff
@@ -318,6 +318,73 @@ def test_nested_field_with_property(spec_fixture):
     }
 
 
+def test_field2property_pluck_spec(spec_fixture):
+    breed = fields.Pluck(CategorySchema, "breed")
+
+    spec_fixture.spec.components.schema("Category", schema=CategorySchema)
+    unplucked = get_schemas(spec_fixture.spec)["Category"]["properties"]["breed"]
+
+    assert spec_fixture.openapi.field2property(breed) == unplucked
+
+
+def test_field2property_pluck_spec_metadatas(spec_fixture):
+    breed = fields.Pluck(
+        CategorySchema,
+        "breed",
+        metadata={
+            "description": "Category breed",
+            "invalid_property": "not in the result",
+            "x_extension": "A great extension",
+        },
+    )
+    result = spec_fixture.openapi.field2property(breed)
+
+    spec_fixture.spec.components.schema("Category", schema=CategorySchema)
+    unplucked = get_schemas(spec_fixture.spec)["Category"]["properties"]["breed"]
+    assert result == {
+        **unplucked,
+        "description": "Category breed",
+        "x-extension": "A great extension",
+    }
+
+
+def test_field2property_pluck_many(spec_fixture):
+    breed = fields.Pluck(CategorySchema, "breed", many=True)
+
+    ret = spec_fixture.openapi.field2property(breed)
+
+    spec_fixture.spec.components.schema("Category", schema=CategorySchema)
+    unplucked = get_schemas(spec_fixture.spec)["Category"]["properties"]["breed"]
+
+    assert ret["type"] == "array"
+    assert ret["items"] == unplucked
+
+
+def test_pluck_field_with_property(spec_fixture):
+    breed_1 = fields.Pluck(CategorySchema, "breed")
```

Comment:
Perhaps rename to `test_field2property_pluck_field_with_property` for consistency but no big deal. If you don't have time, I can merge and rename later or just let it go. I'm not sure the other tests names are really consistent already.

Reply:
I used the [`test_field2property_nest_*` tests](https://github.com/marshmallow-code/apispec/blob/f64ff702646375120320a09a85753a3be6e4995c/tests/test_ext_marshmallow_field.py#L249-L318) as a starting point, so the naming here is consistent with those tests. :-)

I'll clean this one up and split it into 2 tests (2 cases here are already covered by `test_field2property_pluck_spec` and `test_field2property_pluck_many`)

Reply:
In fact, I've refactored all the new tests into a class for a more concise set of tests.